style.css

```diff
@@ -4,110 +4,96 @@ body {
 }
 
 .menu {
-    
+    display: block;
+    text-align: center;
     width: 1920px;
     height: 1080px;
     background: linear-gradient(0deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), url(bg-images/1.jpg);
 }
 
-.bigsun {
-    position: absolute;
-    top: 365px;
-    left: 900px;
-    color: #FFFFFF;
+.block__menu {
+    padding-top: 347px;
```

Comment:
What if I open it on a small screen? I think it'll go away. Please avoid such crazy values. Better to wrap it in a container, which will position it in the center